manifests/cli/azure.pp

```diff
@@ -0,0 +1,10 @@
+# = Hammer AzureRM plugin
```

Comment:
Can you simplify the whole docs to use `puppet-strings` style?
```puppet
# @summary This installs the AzureRM plugin for Hammer CLI
```

Reply:
Done. Removed the `=== Parameters` section. Since there were no separate params to mention with `@param`.
Does that make sense?